src/ansys/units/dimensions.py

```diff
@@ -19,13 +19,20 @@ class Dimensions:
     ----------
     dimensions : dict, optional
         Dictionary of {``BaseDimensions``: exponent, ...}.
+    copy_from : Dimensions, optional
+        A previous instance of Dimensions.
     """
 
     def __init__(
         self,
         dimensions: dict[ansunits.BaseDimensions, Union[int, float]] = None,
+        copy_from: ansunits.Dimensions = None,
```

Comment:
It's worth considering swapping the order of arguments here. That would make it more _Pythonic_ (consider that the argument order is reveresed in the `dict` instantiation below).